### ISSUE TITLE
Chapter 1 Exercise 1.5C

### ISSUE THREAD

**Issue by bsknblc** [gaurangsaini/sipser-computation-3rd-solutions#2]
Hello, 
I think yellow ones are supposed to be the accept states,
![1](https://user-images.githubusercontent.com/60985930/139650591-bc740b8b-a0c8-4905-b48d-d9c270d1c402.PNG)
And the complement of that is should be like this.
![2](https://user-images.githubusercontent.com/60985930/139650596-6f6e36af-f05f-4135-ac1e-526883f5a072.PNG)

**Comment by slow-down** [comment]
no, you can't just accept that for example a is in L because its obv. not in L. 
w in L contains either the substring ab or ba, so only those are the accepted states.
